src/composables/useActions.ts

```diff
@@ -40,10 +41,11 @@ export function useActions() {
     executionHash: string,
     title: string,
     body: string,
-    discussion: string
+    discussion: string,
+    transactions: Transaction[]
   ) {
     if (!web3.value.account) return await forceLogin();
-    const pinned = await pin({ title, body, discussion });
+    const pinned = await pin({ title, body, discussion, transactions });
```

Comment:
Sorry just noticed but could we rename `transactions` to `execution` in draft and pinned file?
```suggestion
    const pinned = await pin({ title, body, discussion, execution });
```